src/content/zero-knowledge-proofs.md

```diff
@@ -0,0 +1,178 @@
+
+ZK-STARK is an acronym for **Zero-Knowledge Scalable Transparent Argument of Knowledge**. If you notice, a ZK-STARK is similar to a ZK-SNARK, except for two qualities:
+
+- **Scalable**: ZK-STARK is faster than ZK-SNARK at generating and verifying proofs when the size of the witness is larger. With STARK proofs, prover and verification times only slightly increase as the witness grows (SNARK prover and verifier times increase linearly with witness size). 
+
+- **Transparent**: ZK-STARK relies on publicly verifiable randomness to generate public parameters for proving and verification instead of a trusted setup. Thus, they are more transparent compared to ZK-SNARKs. 
+
+While ZK-STARK improves on ZK-SNARK, it has a significant drawback: it produces larger proofs. As a result, STARK proofs have higher verification overhead—although this arguably becomes less of an issue when generating proofs for large datasets. In this case, a ZK-STARK may be more cost-effective than a ZK-SNARK. 
+
+## Use-cases for zero-knowledge proofs {#use-cases-for-zero-knowledge-proofs}
+
+### Anonymous payments {#anonymous-payments} 
+
+Credit card payments are often visible to multiple parties, including the payments provider, your bank, and other interested parties (e.g., government authorities). While financial surveillance is good for curbing illegal financial activities, it poses risks for ordinary citizens. Such individuals may wish to keep details of their transactions away from prying eyes. 
+
+Cryptocurrencies were designed to provide a means for users to conduct private, peer-to-peer transactions. But cryptos only offer marginal improvements on financial privacy, as transaction details are publicly visible on the blockchain. Also, user identities are pseudonymous, not completely anonymous, allowing cutting-edge blockchain analysis tools to correlate wallet addresses with real-life identities. 
+
+However, continued research into financial privacy techniques led to the development of “privacy coins”, which are cryptocurrencies specifically designed for completely anonymous transactions. Privacy-focused blockchains, such as ZCash and Monero, shield transaction details, including sender/receiver addresses, asset type, quantity, and the transaction timeline. 
+
+Some will ask, “How can blockchain nodes verify the validity of transactions without accessing transaction details?”. This is where zero-knowledge proofs come into the picture. By baking in zero-knowledge technology into the protocol, privacy-focused blockchain networks allow nodes to validate transactions without needing to access transaction data. 
+
+Zero-knowledge proofs are also being applied to anonymizing transactions on public blockchains. An example is Tornado Cash, a decentralized, non-custodial service that allows users to conduct private transactions on Ethereum. Tornado Cash uses zero-knowledge proofs to obfuscate transaction details and guarantee financial privacy.  
```

Comment:
```suggestion
Zero-knowledge proofs are also being applied to anonymizing transactions on public blockchains. An example is Tornado Cash, a decentralized, non-custodial service that allows users to conduct private transactions on Ethereum. Tornado Cash uses zero-knowledge proofs to obfuscate transaction details and guarantee financial privacy.  Unfortunately, because these are "opt-in" privacy tools they are associated with illicit activity. To overcome this, privacy has to eventually become the default on public blockchains.
```